posthog/hogql/transforms/in_cohort.py

```diff
@@ -354,6 +357,8 @@ def _add_join_for_cohort(
         if must_add_join:
             if is_static:
                 sql = "(SELECT person_id, 1 as matched FROM static_cohort_people WHERE cohort_id = {cohort_id})"
+            elif version is not None:
+                sql = "(SELECT person_id, 1 as matched FROM raw_cohort_people WHERE cohort_id = {cohort_id} AND version = {version})"
```

Comment:
Where is version value supplied for the placeholder?

Reply:
you raise a very good question